plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -91,6 +94,17 @@ private Module getGlueStatisticsModule(Class<? extends GlueColumnStatisticsProvi
                 .in(Scopes.SINGLETON);
     }
 
+    @ProvidesIntoSet
+    @Singleton
+    @ForGlueHiveMetastore
+    public RequestHandler2 createRequestHandler(OpenTelemetry openTelemetry)
+    {
+        return AwsSdkTelemetry.builder(openTelemetry)
+                .setCaptureExperimentalSpanAttributes(true)
```

Comment:
nice!

Can we add similar for Iceberg Glue catalog?

Reply:
This PR added support for Iceberg Glue catalog as well. The above screenshot is captured with Iceberg connector.